ONSdigital/eq-survey-runner
# EQ 13 New Relic integration

**What**
This change install the New Relic agent programmatically. We need to do it this way as we're deploying into Elastic Beanstalk.

**How to test**
1. Create a new relic account
2. Follow the instructions on how to generate a newrelic.ini file - [New Relic](https://docs.newrelic.com/docs/agents/python-agent/getting-started/python-agent-quick-start)
3. Move the file to the top level directory of this repository (or point the EQ_NEW_RELIC_CONFIG_FILE to the location of that file).
4. Run the server
5. Browse to the [health check](http://localhost:5000/healthcheck)
6. Check new relic is receiving information.

**Who can review**
Anyone apart from @warrenmethods
